_data/simple-icons.json

```
@@ -8336,6 +8336,11 @@
             "source": "https://www.patreon.com/brand/downloads",
             "guidelines": "https://www.patreon.com/brand/downloads"
         },
+        {
+            "title": "Payhip",
+            "hex": "3197D6",
+            "source": "https://payhip.com/"
```

```suggestion
            "source": "https://payhip.com"
```
Removal of trailing `/`